deviceclient.py

```diff
@@ -0,0 +1,74 @@
+import paho.mqtt.client as mqtt
+import json
+
+class Notifications:
+	COLORS = (
+		"red"
+		"green"
+		"blue"
+		"yellow"
+		"orange"
+		"white"
+		"cyan"
+		"magenta"
+	)
+
+	_NOTIFICATION_VERSION = 1
+
+	_NOTIFICATION_TEMPLATE = {"record_version": _NOTIFICATION_VERSION, "type":"info", "message":"", "RGB_LED": "white"}
+	
+	_DEFAULT_QoS = 0
+
+	def __init__(self, client):
+		self.mqtt_client = client
+
+	def __str__(self):
+		return "device client used for sending notification messages to IoT device"
+
+	def _send_notification(self, device, payload, qos):
+		self.mqtt_client.publish(device+"/info", payload=json.dumps(payload), qos=self._DEFAULT_QoS)
+
+	def _verify_notification_inputs(self, device, message, RGB_LED, qos):
+		if type(device) != str:
+			raise TypeError("device must be a string")
+		if type(message) != str:
+			raise TypeError("message must be a string")
+		if type(RGB_LED) == list:
+			if len(RGB_LED) != 3:
+				raise TypeError("RGB LED list must only have 3 values")
+			for value in RGB_LED:
+				if type(value) != int:
+					raise TypeError("RGB LED list values must be integers between 0 and 255")
```

Comment:
@AustinDoolittle, what would you recommend here to reduce the number of indents?